getting-started/main.cpp

```diff
@@ -142,9 +142,9 @@ static void sign_a_message_using_rsa(void)
 {
     psa_status_t status;
     psa_key_attributes_t attributes = PSA_KEY_ATTRIBUTES_INIT;
-    uint8_t key[] = RSA_KEY;
+    static uint8_t key[] = RSA_KEY;
```

Comment:
Preexisting, but would be nice to do what you're at it: `key` should be `const`.